src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
+/* eslint-disable no-unused-vars */
 'use strict';
 
-// write code here
+function getSalaryFromData(item) {
+  return parseInt(item.dataset.salary, 10);
+}
+
+function sortList(list) {
+  const items = Array.from(list.children);
+
+  items.sort((a, b) => getSalaryFromData(b) - getSalaryFromData(a));
+  items.forEach((item) => list.appendChild(item));
```

Comment:
Appending items directly to the list without removing them first will result in duplication. Consider using `list.innerHTML = ''` before the `forEach` loop to clear the list, or use `list.replaceChildren(...items)` to replace the children with the sorted items.